CHANGELOG.md

```diff
@@ -25,6 +32,13 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 - Minimum supported Rust version is now 1.36.0.
   ([#1108](https://github.com/nix-rust/nix/pull/1108))
+
+- `Ipv4Addr::octets`, `Ipv4Addr::to_std`, `Error::as_errno`,
+  `ForkResult::is_child`, `ForkResult::is_parent`, `Gid::as_raw`,
+  `Uid::is_root`, `Uid::as_raw`, `Pid::as_raw`, and `PollFd::revents` now take
+  `self` by value.
```

Comment:
What's the motivation for taking `self` by value?  Doesn't that either consume the original value or require Copy?  This change was part of the "Clippy cleanup" commit, which doesn't provide any other context.